README.md

```diff
@@ -1 +1,42 @@
-# webpack-install
\ No newline at end of file
+# Grill & Chill
+
+> The website allows food lovers to look for the food cards by name, see all cards in series, add new likes and comments(yet to be implimented) to the cards. 
+
+## Built With
+
+- Major languages: JavaScript HTML & CSS 
+- linters
+- webpack to bundle JavaScript
+- Frameworks: none
+- API services
+
+[Webpage Link ](https:///)
+
+## Live Demo (if available)
+
+## Authors
+
+👤 **Shubh M**
+
+- GitHub: [@Shubh-Dev](https://github.com/Shubh-Dev)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/shubh.scb)
+
+## 🤝 Contributing
```

Comment:
- [ ] please make sure to use [the template](https://github.com/microverseinc/readme-template) for the readme file provided by Microverse, including the links (for example to the MIT license). keep in mind that all sections of the template are mandatory except for the "Live Demo", "Run Tests" and "Deployment" sections, these are only mandatory if the specific project you're working on requires it.